binaries/apiserver/main.go

```diff
@@ -10,7 +11,9 @@ func main() {
 	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
 	transportFactory := thrift.NewTTransportFactory()
 
-	err := server.Serve(server.NewHandler(), "localhost:9090", transportFactory, protocolFactory)
+	queue, _ := memory.NewSimpleQueue()
+	handler := server.NewHandler(queue)
+	err := server.Serve(handler, "localhost:9090", transportFactory, protocolFactory)
```

Comment:
Should loalhost:9090 be hardcoded.  For instance when this is actually ran in aurora, we don't get to pick the port.  Suggest maybe having a variable for now with a TODO to read from config.